modules/client-configuration/pages/clients-opensuseleap.adoc

```diff
@@ -130,6 +134,30 @@ ifeval::[{uyuni-content} == true]
 
 |===
 
+[cols="1,1,1,1,1,1,1,1", options="header"]
```

Comment:
Having a look at https://w3.suse.de/~rosuna/docu-PRs-uyuni/leap15.5/build/en/uyuni/client-configuration/clients-opensuseleap.html, I don't think you need a separate new table for 15.5, as the list of channels is the same we had for 15.4

Reply:
True, I thought the separation was for reading purposes, but missed the SLE backport channels. Will fix that.